components/Buttons/src/MDCButton.m

```diff
@@ -182,8 +182,8 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
     }
     [self updateBackgroundColor];
 
-    if ([aDecoder containsValueForKey:MDCButtonAccessibilityLabelsKey]) {
```

Comment:
If someone restores from a previous version, this code will drop the title because you've changed the key. You should check for both keys.